Command/DoctrineCommand.php

```diff
@@ -32,7 +32,7 @@ public static function configureMigrations(ContainerInterface $container, Config
     {
         if (!$configuration->getMigrationsDirectory()) {
             $dir = $container->getParameter('doctrine_migrations.dir_name');
-            if (!@mkdir($dir, 0777, true) && !is_dir($dir)) {
+            if (!is_dir($dir) && !@mkdir($dir, 0777, true)) {
```

Comment:
The `!is_dir()` check after the `mkdir()` should be kept. It prevents throwing an exception in case the directory was created in the meantime by another process.